### PR TITLE
Update Dapr quickstart

### LINKED ISSUE #7: "Dapr microservices tutorial" doesn't provide a 60k level overview of the deployment early in the description.
As someone who is trying to learn a new subject, it is helpful to understand what I will be doing in this tutorial, seeing the whole picture first and then breaking into steps helps learners in the process of understanding.

If we can have a visual that shows the full deployment and a small text that explains at a high level what the deployment is implementing that would greatly reduce the "time to learn" (TTL) metric

Page Reference : https://edge.radapp.dev/user-guides/tutorials/dapr-microservices/overview/

### LINKED ISSUE #9: The lesson/learning objectives of the Dapr Microservices Tutorial are not explicitly listed.
**Observed behavior**

As a developer exploring Radius, I find myself debating if I should do the "Dapr Microservices Tutorial" because the benefits/gains and motivation for going through it are not clearly listed.

I went ahead and checked online to see if there is a logical explanation to this feeling and found this site: [8 Best Practices To Create Effective Online Training Tutorials](https://elearningindustry.com/best-practices-create-effective-online-training-tutorials)

It offers great motivation as a best practice for tutorials to "Highlight The Benefits Up Front"

> 3. Highlight The Benefits Up Front
> Give employees a general overview of the key takeaways, benefits, and real-world applications of the online training tutorial right from the start. In fact, you may want to reiterate these key points throughout the online training tutorial. Include a timeline that shows their progress, and explain the "why" behind each step. For example, the second step in the compliance online training tutorial contains the hazardous spill. This helps to ensure the well-being of others while the individual cleans up the contaminant. Reiterating the benefits also improves employee motivation and engagement. This is due to the fact that they know the online training tutorial ties into their workplace performance. It offers them useful knowledge and skills they can use to achieve their professional goals.

**Desired behavior**
More content should exist in the Overview section of the tutorial that lists the motivation and key lessons that will be covered as part of the tutorial and their relevance to understand Radius and its concepts.

**OS/rad version** - N\A

**Screenshots** - N\A

**Additional context** 

Page Reference: https://edge.radapp.dev/user-guides/tutorials/dapr-microservices/

### LINKED ISSUE #10: Dapr Microservices tutorial doesn't require prior knowledge of Dapr however doesn't provide an overview of Dapr and relevance to the tutorial
**Observed behavior**
In the "Overview" section of the "Dapr Microservices Tutorial" the website says:
> No prior knowledge of Dapr, Radius, or Bicep is needed.

This sets an implicit expectation that there will be a high level overview of these concepts and their relevance in the attainment of the objectives of the tutorial but this is overlooked/omitted from the text.

**Desired behavior**
* A high level overview of the concepts (Dapr, Bicep) and their relevance in the attainment of the objectives of the tutorial 
* Links to learn more about these if necessary

**OS/rad version**
N\A

**Screenshots**

N\A

**Additional context**

### LINKED ISSUE #118: Make a new Dapr Quickstart
when we do this change from a tutorial to a quickstart, we also need to check that all the verbiage in docs matches the bicep file. 
looks like there a mismatch between resource names at least
https://github.com/project-radius/docs/blob/edge/docs/content/getting-started/quickstarts/dapr-quickstart/dapr-microservices.bicep
vs. 
https://edge.radapp.dev/getting-started/quickstarts/dapr-quickstart/overview/
vs.
https://github.com/project-radius/docs/blob/edge/docs/content/getting-started/quickstarts/dapr-quickstart/code/dapr.bicep

...and there's no frontend resource in this one in v0.11...
https://github.com/project-radius/docs/blob/v0.11/docs/content/user-guides/tutorials/dapr-microservices/code/dapr.bicep

Overall, we need a bigger effort to make sure that people working on tutorials & quickstarts & samples can all get to the FULL example bicep template very easily. Current state seems like a great way to accidentally waste people's time.

### LINKED ISSUE #131: Add Dapr sidecars and a Dapr statestore to the app
When completing the Tutorial - I could not get the 'no orders yet' message to pop up.  I would always get 'Orders state store not specified' - looking through the documentation, there is no mention of adding an env: { } section in the backend container, and checking the 'known good' file, it does not contain one either.  

when digging into this, I could see (through some kubectl commands) that there was not an environment variable for CONNECTION_ORDERS_STATESTORENAME - only CONNECTION_STATESTORE_STATESTORENAME=orders but seems the image was looking for the CONNECTION_ORDERS_STATESTORENAME=orders - so I added it manually, and it fixed the problem.   The documentation probably should be fixed

I was able to resolve the issue and get the 'no orders yet' message by: 

adding the following section under the backend resource container section: 

   env: {
     CONNECTION_ORDERS_STATESTORENAME: 'orders'
   }

### LINKED ISSUE #229: Add Dapr - Quickstarts resource list extra columns
**Observed behavior**
On executing rad resource list --application dapr-tutorial. the output has resource and type columns only.

**Desired behavior**
Docs should be updated to inlcude only resource and type columns. Remove the additional columns

**OS/rad version**
rad version
RELEASE     VERSION      BICEP     COMMIT
0.12.0-rc1  v0.12.0-rc1  0.7.14    484ae03c8cd1dc7bb7a206836d4cd4adef18fc0d

**Screenshots**
Screen Shot 2022-07-22 at 12 03 42 PM

**Additional context**
link to page: https://edge.radapp.dev/getting-started/quickstarts/dapr-quickstart/add-dapr/

### PR DESCRIPTION
**Pull request by AaronCrawfis**
Closes #131 
Closes #118 
Closes #229 
Closes #7

### CODE REVIEW

File: docs/content/getting-started/quickstarts/dapr-quickstart/dapr-microservices-overview/index.md [hunk 1]
```diff
@@ -39,6 +38,12 @@ The `backend` container is configured with a [Dapr sidecar extension]({{< ref da
 
 <img src="./backend.png" alt="A diagram of the backend order processing service" width=600 />
 
+## Connector
```
Comment (emily-potyraj) [inline docs/content/getting-started/quickstarts/dapr-quickstart/dapr-microservices-overview/index.md]: Can you please expand a little bit more here? Maybe about 
- a little more explanation of the image below: are you using just Azure Table Storage? or both ATS and Redis Cache? 
- how Dapr brings code-level portability and is a great combo with the infra-level portability that Radius brings

File: docs/content/getting-started/quickstarts/dapr-quickstart/_index.md [hunk 2]
```diff
@@ -28,12 +28,10 @@ In this tutorial, you will:
 
 ## Prerequisites
```
Comment (emily-potyraj) [inline docs/content/getting-started/quickstarts/dapr-quickstart/_index.md:37]: Do you want to add a quick blurb here about the value prop of Dapr + Radius? I feel like the quickstart can ramp them up on both the How To and touch on the Why for orientation.

File: docs/content/getting-started/quickstarts/dapr-quickstart/dapr-microservices-overview/index.md [hunk 3]
```diff
@@ -53,7 +58,7 @@ Radius offers communication between services via [Routes]({{< ref "appmodel-conc
 
 ### Dapr service invocation
 
-In this tutorial we will be using a [`dapr.io.InvokeHttpRoute`]({{< ref dapr-http >}}) resource to model communication from `frontend` to `backend`. This allows `frontend` to use Dapr service invocation to interact with `backend`.
+In this tutorial we will be using a [Dapr HTTP invoke route]({{< ref dapr-http >}}) resource to model communication from `frontend` to `backend`. This allows `frontend` to use Dapr service invocation to interact with `backend`.
 
 <img src="./invoke.png" alt="A diagram of the Dapr service invocation" width=500 />
```
Comment (emily-potyraj) [inline docs/content/getting-started/quickstarts/dapr-quickstart/dapr-microservices-overview/index.md:62]: This Summary section feels repetitive to the summary section on the quickstart's main index page
("This tutorial will teach you how to deploy a Dapr microservices application using Radius. You will learn:")

What do you think about removing the summary section on this page and pushing any info you think the user needs from this paragraph into the paragraph on the main page?

File: docs/content/getting-started/quickstarts/dapr-quickstart/dapr-microservices-initial-deployment/index.md [hunk 4]
```diff
@@ -9,7 +9,7 @@ weight: 2000
 
 ## Create a Bicep file
 
-Radius uses the [Bicep language]({{< ref bicep >}}) as its file-format and structure. In this tutorial you will define an app named `dapr-tutorial` that will contain the containers, statestore, and route resources, all described in Bicep.
+Radius uses the [Bicep language]({{< ref bicep >}}) as its file-format and structure. In this tutorial you will define an app named `dapr-quickstart` that will contain the containers, statestore, and route resources, all described in Bicep.
```
Comment (emily-potyraj) [inline docs/content/getting-started/quickstarts/dapr-quickstart/dapr-microservices-initial-deployment/index.md]:
```suggestion
Radius uses the [Bicep language]({{< ref bicep >}}) as its file-format and structure. In this tutorial you will define an app named `dapr-quickstart` that will describe the container, statestore, and route resources in Bicep.
```
suggestion for readability

File: docs/content/getting-started/quickstarts/dapr-quickstart/dapr-microservices-initial-deployment/snippets/app.bicep [hunk 5]
```diff
@@ -4,7 +4,7 @@ param location string = resourceGroup().location
 param environment string
```
Comment (emily-potyraj) [inline docs/content/getting-started/quickstarts/dapr-quickstart/dapr-microservices-initial-deployment/snippets/app.bicep:4]: can't comment on line 3, but I think `location` needs to be updated to = 'global'
that change likely needs to be made across all the snippets for this quickstart, right?

File: docs/content/getting-started/quickstarts/dapr-quickstart/dapr-microservices-initial-deployment/index.md [hunk 6]
```diff
@@ -55,17 +55,17 @@ Now you are ready to deploy the application for the first time.
    backend    Container
```
Comment (emily-potyraj) [inline docs/content/getting-started/quickstarts/dapr-quickstart/dapr-microservices-initial-deployment/index.md]:
```suggestion
   backend   Applications.Core/containers
```

File: docs/content/getting-started/quickstarts/dapr-quickstart/dapr-microservices-add-dapr/index.md [hunk 7]
```diff
@@ -91,10 +83,10 @@ There is a known issue where deployments to Azure will fail with a "NotFound" er
 1. You can confirm all the resources were deployed by running:
 
    ```sh
-   rad resource list --application dapr-tutorial
+   rad resource list --application dapr-quickstart
    ```
 
-   You should see both `backend` and `statestore` components in your `dapr-tutorial` application. Example output:
+   You should see both `backend` and `statestore` components in your `dapr-quickstart` application. Example output:
```
Comment (emily-potyraj) [inline docs/content/getting-started/quickstarts/dapr-quickstart/dapr-microservices-add-dapr/index.md:90]: there are 5 resources deployed at this point.
output from the above command:
```
RESOURCE         TYPE
statestore       applications.connector/daprstatestores
backend-route    applications.connector/daprinvokehttproutes
redis-route      applications.core/httproutes
redis-container  applications.core/containers
backend          applications.core/containers
```

File: docs/content/getting-started/quickstarts/dapr-quickstart/dapr-microservices-add-dapr/index.md [hunk 8]
```diff
@@ -105,7 +97,7 @@ There is a known issue where deployments to Azure will fail with a "NotFound" er
 1. To test out the state store, open a local tunnel on port 3000 again:
 
    ```sh
-   rad resource expose Container backend --application dapr-tutorial --port 3000
+   rad resource expose Container backend --application dapr-quickstart --port 3000
    ```
 
 1. Visit the the URL [http://localhost:3000/order](http://localhost:3000/order) in your browser. You should see the following message:
```
Comment (emily-potyraj) [inline docs/content/getting-started/quickstarts/dapr-quickstart/dapr-microservices-add-dapr/index.md:104]: you have json format for the example output below, but the previous page has example output with no format (just '```' as the header)

File: docs/content/getting-started/quickstarts/dapr-quickstart/dapr-microservices-add-ui/index.md [hunk 9]
```diff
@@ -47,7 +49,7 @@ services.AddSingleton<HttpClient>(DaprClient.CreateInvokeHttpClient(appId));
 
    ```sh
    Public Endpoints:
-      Gateway            frontend-gateway           IP-ADDRESS
+    gateway         Applications.Core/gateways http://gateway.dapr-quickstart.<IP>.nip.io
```
Comment (emily-potyraj) [inline docs/content/getting-started/quickstarts/dapr-quickstart/dapr-microservices-add-ui/index.md:52]: `http://gateway.dapr-quickstart.<IP>.nip.io` - what is this?

Comment (AaronCrawfis) [reply docs/content/getting-started/quickstarts/dapr-quickstart/dapr-microservices-add-ui/index.md:52]: Gateways will automatically generate a hostname based on nip.io (An IP to DNS resolver service) for the user if a hostname was not specified in the definition. More info here: https://edge.radapp.dev/reference/resource-schema/core-schema/gateway/#hostname-generation

File: docs/content/getting-started/quickstarts/dapr-quickstart/dapr-microservices-overview/index.md (hunk 3 above)
Comment (AaronCrawfis) [reply docs/content/getting-started/quickstarts/dapr-quickstart/dapr-microservices-overview/index.md:62]: Love it. Will update

File: docs/content/getting-started/quickstarts/dapr-quickstart/dapr-microservices-add-ui/index.md (hunk 9 above)
Comment (emily-potyraj) [reply docs/content/getting-started/quickstarts/dapr-quickstart/dapr-microservices-add-ui/index.md:52]: thanks. should we add a link out to that page here?

File: docs/content/getting-started/quickstarts/dapr-quickstart/dapr-microservices-add-dapr/index.md (hunk 7 above)
Comment (AaronCrawfis) [reply docs/content/getting-started/quickstarts/dapr-quickstart/dapr-microservices-add-dapr/index.md:90]: This has been fixed by switching to Kubernetes resources

File: docs/content/getting-started/quickstarts/dapr-quickstart/dapr-microservices-add-ui/index.md (hunk 9 above)
Comment (AaronCrawfis) [reply docs/content/getting-started/quickstarts/dapr-quickstart/dapr-microservices-add-ui/index.md:52]: Added a note to learn more with a link to the section on hostname generation

File: docs/content/getting-started/quickstarts/dapr-quickstart/dapr-microservices-add-dapr/index.md (hunk 8 above)
Comment (AaronCrawfis) [reply docs/content/getting-started/quickstarts/dapr-quickstart/dapr-microservices-add-dapr/index.md:104]: Added json formatting code to previous page. Good catch!

**Pull request merged**